pkg/provider/azure_standard.go

```diff
@@ -292,11 +292,11 @@ func (az *Cloud) getloadbalancerHAmodeRuleName(service *v1.Service) string {
 }
 
 func (az *Cloud) getSecurityRuleName(service *v1.Service, port v1.ServicePort, sourceAddrPrefix string) string {
+	safePrefix := strings.Replace(sourceAddrPrefix, "/", "_", -1)
+	safePrefix = strings.Replace(safePrefix, ":", "_", -1) // Consider IPv6 address
```

Comment:
Is it possible to get same rule name with different IPs:
e.g. both "fe80::1/32" and "fe80::1:32" could be translated to "fe80__1_32"

Reply:
Currently, such problem won't happen because `sourceAddrPrefix` is an IPNet and it has a mask.
However, `.` can be used instead because it does no harm and it is possible someday in the future, someone changes `sourceAddrPrefix` but forget to update here. Thank you.